### ISSUE TITLE
PHPCS doesn't check files that begin with a dot

### ISSUE THREAD

**Issue by w0rp** [squizlabs/PHP_CodeSniffer#1928]
This bug was originally raised in ALE here: https://github.com/w0rp/ale/issues/1408

I installed PHP_CodeSniffer 3.23 and created a simple file.

```php
<?php
$x = 1
?>
```

If you name the file `foo.php`, and run `phpcbf -q --stdin-path=foo.php  - < foo.php`, then `phpcbf` returns the contents of the file. If you rename the file to `.foo.php` and run `phpcbf -q --stdin-path=.foo.php  - < .foo.php`, then `phpcbf` returns the following.

```
No fixable errors were found

Time: 25ms; Memory: 4Mb
```

ALE wants to be able to pipe the buffer data into the program and read the fixed results from stdout, so this causes some issues where buffers are replaced with the above output.

**Comment by gsherwood** [comment]
Note that this isn't an issue with PHPCBF. PHPCS itself will not check a hidden file, so it doesn't produce any error messages for PHPCBF to fix. The same thing would happen if you didn't specify an extension on the stdin-path.

Having PHPCS start checking hidden files, and/or those without an extension would be a fairly big change.

**Comment by w0rp** [comment]
I didn't know that hidden files were special. I'll make ALE skip checking any files which start with a dot for now.

**Comment by gsherwood** [comment]
Closing as I've had not further interest in making a core change like this, and it couldn't be made until a major release anyway. If more interest comes up, I'll revisit for version 4.
